### PR TITLE
pdfpc 4.6.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/pdfpc.rb [hunk 1]
```diff
@@ -18,26 +18,21 @@ class Pdfpc < Formula
...
-
   depends_on "cmake" => :build
   depends_on "vala" => :build
+  depends_on "discount"
   depends_on "gst-plugins-good"
   depends_on "gtk+3"
+  depends_on "json-glib"
   depends_on "libgee"
   depends_on "librsvg"
+  depends_on "libsoup@2"
```
Comment (chenrui333) [inline Formula/pdfpc.rb:29]: raise an issue for this in the upstream, https://github.com/pdfpc/pdfpc/issues/664

**Comment by danielnachun** [comment]
It looks like this now has a build dependency on `pkg-config` on Linux to find at least `discount` and possible other packages:
```
/home/linuxbrew/.linuxbrew/Cellar/cmake/3.25.1/share/cmake/Modules/FindPkgConfig.cmake:829 (_pkg_check_modules_internal)
```

File: Formula/pdfpc.rb [hunk 2]
```diff
@@ -18,34 +18,30 @@ class Pdfpc < Formula
...
-    depends_on "qrencode"
-
-    on_linux do
-      depends_on "webkitgtk"
-    end
-  end
-
   depends_on "cmake" => :build
   depends_on "vala" => :build
+  depends_on "discount"
```
Comment (SMillerDev) [inline Formula/pdfpc.rb:23]:
```suggestion
  depends_on "pkg-config" => :build
  depends_on "discount"
```

**Comment by chenrui333** [comment]
still `libmarkdown` issue on linux build
```
  -- Checking for module 'libmarkdown'
  --   No package 'libmarkdown' found
```

**Comment by danielnachun** [comment]
I did some more local testing and there were multiple other problems but I now have a working solution:

1. The reason `libmarkdown` still cannot be found is because the `discount` formula is missing both the `pkg-config` files and the shared libraries.  I will have to open a separate PR to fix this and we'll have to rebase this once that fix is merged.
2. I was able to fix the Markdown support in Linux by slightly patching the `CMakeLists.txt` to look for `webkit2gtk-4.1` instead of `webkitgtk-4.0`.

**Comment by chenrui333** [comment]
> 1. The reason `libmarkdown` still cannot be found is because the `discount` formula is missing both the `pkg-config` files and the shared libraries.  I will have to open a separate PR to fix this and we'll have to rebase this once that fix is merged.

Sounds good, feel free to reopen this PR once that gets fixed.

**Comment by chenrui333** [comment]
#155471

**Pull request closed without merging**